src/test/integrasjonstester/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/StegServiceTest.kt

```diff
@@ -50,6 +56,9 @@ class StegServiceTest : OppslagSpringRunnerTest() {
 
     @BeforeEach
     fun setup() {
+        every { registerPersonGrunnlagSteg.utførSteg(any()) } returns Unit
```

Comment:
Som @UyQuangNguyen fortalte meg så kan du bruke `just runs` istedenfor `returns Unit` her.
```suggestion
        every { registerPersonGrunnlagSteg.utførSteg(any()) } just runs
```